Constitution.md

```diff
@@ -171,17 +171,26 @@ The Committee may appoint as Honorary President and Vice-President, any persons
 3. An Extraordinary General Meeting (EGM) may be called by 5% of the membership of the Society by application to the Secretary. An EGM shall be called as specified on the application, but at least 7 days’ notice shall be given.
 4. Quorum of such meetings shall be set at twice the Committee size, plus one.
-5. Only full members, not Friends of the Society, should be counted during any AGM or EGM or in the process of calling for an EGM
+5. Only full members, not Friends of the Society, are to be considered "eligible voters". Only eligible voters should be counted during any AGM or EGM or in the process of calling for an EGM.
 
 ## Elections
 
 1. Elections shall be held at the Annual General Meeting to elect officers of the Committee for the following session.
 2. There shall be provision for by-elections for positions unfilled at the AGM to be held at an EGM in October.
 3. An EGM must be called if the held Committee positions are not in line with Bristol SU regulations.
-4. Any restrictions on who can stand must be stated –note that such restrictions must not contravene the Code of Conduct.
+4. Any restrictions on who can stand must be stated – note that such restrictions must not contravene the Code of Conduct.
 5. Nominations must be made in accordance with instruction officially communicated by the Committee.
-6. The voting system shall be a single vote for each position, with a simple majority.
-7. The Chairperson of the General Meeting to have the casting vote in the event of a tie, having not voted previously.
+6. Voting must be by secret, secure ballot either through paper or electronic means.
+7. The default system of voting shall be First Past the Post ("FPTP"), for ease of counting only. Eligible voters shall cast (up to) the same number of votes as there are positions to be filled. Each vote shall be a mark next the selected candidate(s). In a multi-position election, each voter may not vote more than once for any candidate. The number of votes received for each candidate is counted, and the highest-polling candidate(s) corresponding to the number of positions to be filled are elected.
+8. However, it is recognized that the Single Transferable Vote ("STV") system of voting is typically a fairer way of electing candidates, though requires a more intensive counting process. If one or more of the following apply prior to the commencement of each election, then STV must be used for that election and it shall not be permitted to elect that position on the basis of an election conducted using the FPTP system of voting. These criteria shall apply on a per-position basis, so different systems of voting may be used for each election within one General Meeting. It shall not be permitted to change retrospectively the system of voting after the commencement of that election. There shall be no limit as to the number of elections conducted using either system of voting, provided the criteria for doing so are duly observed.
```

Comment:
"and it shall not be permitted to elect that position on the basis of an election conducted using the FPTP system of voting" seems overly verbose. "instead of FPTP" would do the trick.